one/src/daemon.rs

```diff
@@ -178,6 +180,41 @@ pub struct DaemonOpts {
         requires = "experimental_features"
     )]
     flight_sql_bind_address: Option<String>,
+
+    /// Remote anchor service URL
+    #[arg(long, env = "CERAMIC_ONE_REMOTE_ANCHOR_SERVICE_URL")]
+    remote_anchor_service_url: Option<String>,
+
+    /// Ceramic One anchor interval in seconds
+    #[arg(long, default_value_t = 3600, env = "CERAMIC_ONE_ANCHOR_INTERVAL")]
+    anchor_interval: u64,
+
+    /// Ceramic One anchor batch size
+    #[arg(
+        long,
+        default_value_t = 1_000_000,
+        hide = true,
+        env = "CERAMIC_ONE_ANCHOR_BATCH_SIZE"
+    )]
+    anchor_batch_size: u64,
+
+    /// Ceramic One anchor polling interval in seconds
```

Comment:
can you add a little context on what polling does? Answer the question of why I want a short or longer interval. Also its not clear how this interval is different from the anchor_interval above.